README.md

```diff
@@ -64,6 +66,18 @@ The file `rust-mode-tests.el` contains tests that can be run via
 You can use `run_rust_emacs_tests.sh` to run them in batch mode, if
 you set the environment variable EMACS to a program that runs emacs.
 
+## Features
+
+### Formatting with [rustfmt][rfmt]
+
+The `rust-format-buffer` function will format your code with
+[rustfmt][rfmt] if installed.
```

Comment:
That would be appreciated. Also, it would be interesting to know how to automatically call rustfmt on save. But this might be too subjective?